src/app/container/modal/pick-unit.js

```diff
@@ -79,5 +84,5 @@ const mapDispatchToProps = {
 export default compose(
   connect(mapStateToProps, mapDispatchToProps),
   withState('unit', 'setUnit', props => props.globalUnit),
-  withState('sameMaterials', 'setSameMaterials', 'yes') // Use `yes` as a default
+  withState('sameMaterials', 'setSameMaterials', props => props.useSameMaterial && 'yes')
```

Comment:
Der Datentyp hier ist jetzt inkonsistent: 
```suggestion
  withState('sameMaterials', 'setSameMaterials', props => props.useSameMaterial ? 'yes' : 'no')
```

Reply:
Ja stimmt, das wurde ein bisschen schwammig gelöst ;) Ich habe einen neue Lösung ge-pusht, welche mit recompose das Anzeigen der Buttons löst.